content/company-info-and-process/communication/code_of_conduct.md

```diff
@@ -91,15 +91,15 @@ Our success depends on the trust we earn from customers, partners, and each othe
 
 We provide company resources for company business use.
 
-Personal use on company equipment such as laptops is fine as long as your use does not impede your work, disrupt our work environment, or break any laws. See our [Device usage and privacy policy](../../../departments/tech-ops/process/team_device_usage_privacy).
+Personal use on company equipment such as laptops is fine as long as your use does not impede your work, disrupt our work environment, or break any laws. See our [Device usage and privacy policy](../../../departments/tech-ops/process/team_device_usage_privacy.md).
 
 Any technology that you buy using Sourcegraph money or that you use to do work for Sourcegraph is subject to access by Sourcegraph at any time and subject to discovery in the event of litigation involving the company. Be mindful of what you put on your company equipment. Avoid putting any personal content that you would like to keep private.
 
 Direct questions about the proper use of company resources to the Tech Ops Team at tech-ops@sourcegraph.com.
 
 ### Protect confidential information of Sourcegraph, its customers, users, and other business partners
 
-We are an [open and transparent company](../../about-sourcegraph#sourcegraph-open-product-open-company-open-source). However, you still have a duty to protect confidential company information as well as nonpublic information entrusted to you by Sourcegraph employees, customers, and other business partners. Confidential and proprietary information includes nonpublic pricing, financial data, and customer information. See our [level of transparency chart](../../values#open-and-transparent).
+We are an [open and transparent company](../../about-sourcegraph#sourcegraph-open-product-open-company-open-source.md). However, you still have a duty to protect confidential company information as well as nonpublic information entrusted to you by Sourcegraph employees, customers, and other business partners. Confidential and proprietary information includes nonpublic pricing, financial data, and customer information. See our [level of transparency chart](../../values#open-and-transparent.md).
```

Comment:
This is a confusing one... When there's a `#...` part of the URL, that actually should come after the full filename. So the `.md` here needs to come after `values` and before `#`

Reply:
```suggestion
We are an [open and transparent company](../../about-sourcegraph.md#sourcegraph-open-product-open-company-open-source). However, you still have a duty to protect confidential company information as well as nonpublic information entrusted to you by Sourcegraph employees, customers, and other business partners. Confidential and proprietary information includes nonpublic pricing, financial data, and customer information. See our [level of transparency chart](../../values.md#open-and-transparent).
```